dbms/src/Storages/DeltaMerge/tests/gtest_dm_vector_index.cpp

```diff
@@ -535,38 +535,32 @@ try
     dm_file = restoreDMFile();
     auto index_infos = std::make_shared<LocalIndexInfos>(LocalIndexInfos{
         // index with index_id == 3
-        LocalIndexInfo{
-            .kind = TiDB::ColumnarIndexKind::Vector,
-            .index_id = 3,
-            .column_id = vec_column_id,
-            .def_vector_index = std::make_shared<TiDB::VectorIndexDefinition>(TiDB::VectorIndexDefinition{
+        LocalIndexInfo(
+            3,
+            vec_column_id,
+            std::make_shared<TiDB::VectorIndexDefinition>(TiDB::VectorIndexDefinition{
```

Comment:
Does this change really improve readability? (And it does not make anything short as well)

Reply:
To avoid something like:

```cpp
LocalIndexInfo{
            .kind = TiDB::ColumnarIndexKind::Inverted,
            .index_id = 3,
            .column_id = vec_column_id,
            .def_vector_index = std::make_shared<TiDB::VectorIndexDefinition>(TiDB::VectorIndexDefinition{
                .kind = tipb::VectorIndexKind::HNSW,
                .dimension = 3,
                .distance_metric = tipb::VectorDistanceMetric::L2,
            })
},
```